specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/machineLearningServices.json

```diff
@@ -2875,6 +2879,18 @@
         "isLatestOsImageVersion": {
           "type": "boolean",
           "description": "Specifies whether this compute instance is running on the latest operating system image."
+        },
+        "rebootPending": {
+          "type": "boolean",
+          "description": "Specifies whether this compute instance is pending for reboot to finish os patching."
+        },
+        "patchStatus": {
```

Comment:
What would be the status when there is no patch?

Reply:
will be null